cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -103,7 +105,14 @@ func runCleanupNode(c workflow.RunData) error {
 	if certsDir != kubeadmapiv1.DefaultCertificatesDir {
 		klog.Warningf("[reset] WARNING: Cleaning a non-default certificates directory: %q\n", certsDir)
 	}
+
 	dirsToClean = append(dirsToClean, certsDir)
+	if r.CleanupTmpDir() {
+		// It's possible the tmp dir is overridden by Env, but kubeadm is unable to know how that Env is defined
```

Comment:
is this comment accurate though? we are using hardcoded constants to form the path.

Reply:
I am thinking we might let other `cmd` able to redirect the tmp directory as well, for consistency.  WDYT?

Reply:
the tmp dir for dryrun is not the tmp dir for backup, they are different concepts. the dry run dir ends up in /tmp, while the other tmp dir is under /kubernetes.

that env override was only added to make integration tests happy for init. preferably we should not add similar overrides for more commands.

Reply:
Am I miss something? If the env is not defined, dryrun will use the /etc/kubernetes/tmp, right?

```
ls /etc/kubernetes/tmp/kubeadm-init-dryrun328586547
admin.conf                    ca.crt                   front-proxy-ca.key            kube-scheduler.yaml
apiserver.crt                 ca.key                   front-proxy-client.crt        sa.key
...
```

https://github.com/kubernetes/kubernetes/blob/7f3f39b2051fa5f4f22f1449f9ef632de5172908/cmd/kubeadm/app/constants/constants.go#L596-L600

Reply:
But anyway, now that this env is only added for integration test, I agree that comments is not necessary.

Reply:
>  If the env is not defined, dryrun will use the /etc/kubernetes/tmp, right?

ok, you are right.

i think we can drop the comment.